portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,32 +37,37 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comments").addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comments").addSort("timestamp", SortDirection.DESCENDING).addSort("username");
```

Comment:
Your `Comment` class doesn't look like it has a timestamp. Is that something you want to add?